src/components/molecules/TableHeader/TableHeader.tsx

```diff
@@ -213,13 +195,12 @@ const TableHeader: React.FC<TableHeaderProps> = ({
 
         {tableCapacity && (
           <span className="table-header__seats-left">
-            {numberOfSeatsLeft} seats left
+            {tableOfUser && tableOfUser.subtitle && (
+              <label>{tableOfUser.subtitle} -</label>
+            )}{" "}
```

Comment:
Is this the intent?

```suggestion
            {tableOfUser?.subtitle && (
              <label>{tableOfUser.subtitle} -</label>
            )}{" "}
```

Reply:
Yes, the design now shows `description - maxseats` Ex: `My awesome table - 10 seats`

Reply:
Description is optional, and `description -` is rendered conditionally

Reply:
The new design says `description` and `topic` but the old implementation is `title` and `subtitle` this is very annoying and confusing when talking in a conversation like this about it. Not sure, how to proceed and if I should rename the 2 fields in the table object.

Reply:
Renaming the table object that is used in scripts/existing backend/data is kind off the cards. Way too big of a scope creep IMO.

Any props in our code should probably use the new naming I guess. And maybe add `@debt` comments to the firestore data model explaining what their new names there are.

Reply:
The fields were being used only in these 2-3 components, so I renamed them. Fixed in: https://github.com/sparkletown/sparkle/pull/1377/commits/4b7532e6f02068126c212a6f0dab1714aaaad55a